### PR TITLE
Update previews_configuration.adoc

### PR DESCRIPTION
**Pull request by jnweiger**
Preview generator generator has both input and output limits.  Adding input.

Backport to 10.14 please.

### CODE REVIEW

**Review by phil-davis — COMMENTED** [review]
Text LGTM .
`preview_max_dimensions` also gets documented in:
https://github.com/owncloud/docs-server/blob/master/modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc

Do we want to also link to that?
(but I see that none of these config options have a link across to the config_sample_php_parameters doc)

File: modules/admin_manual/pages/configuration/files/previews_configuration.adoc [hunk 1]
```diff
@@ -192,6 +192,20 @@ The example below adds the preview provider for `SGI` and `HEIC` images:
...
+The default value is defined in `config/config.sample.php` as `6016x4000` which is sufficient for 24MP landscape images.
+
+The following example would change the limit so that 24MP portrait images also feature preview generation:
+
+[source,php]
+----
+  'preview_max_dimensions' => '6016x6016',
+----
+
+NOTE: The dimensions here limit the size of the original image (input to the preview generator). The settings below limit the size of the generated preview images (output).
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/files/previews_configuration.adoc:207]:
```suggestion
NOTE: The dimensions here limit the size of the original image (input to the preview generator). The settings below limit the size of the generated preview images (output). For more details see the xref:configuration/server/config_sample_php_parameters.adoc#define-the-maximum-dimensions-of-the-original-image-for-preview-generation[Define the maximum dimensions of the original image for preview generation].
```

File: modules/admin_manual/pages/configuration/files/previews_configuration.adoc [hunk 2]
```diff
@@ -192,6 +192,20 @@ The example below adds the preview provider for `SGI` and `HEIC` images:
...
+The default value is defined in `config/config.sample.php` as `6016x4000` which is sufficient for 24MP landscape images.
+
+The following example would change the limit so that 24MP portrait images also feature preview generation:
+
+[source,php]
+----
+  'preview_max_dimensions' => '6016x6016',
+----
+
+NOTE: The dimensions here limit the size of the original image (input to the preview generator). The settings below limit the size of the generated preview images (output). For more details see the xref:configuration/server/config_sample_php_parameters.adoc#define-the-maximum-dimensions-of-the-original-image-for-preview-generation[Define the maximum dimensions of the original image for preview generation].
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/files/previews_configuration.adoc:207]:
```suggestion
NOTE: The dimensions here limit the size of the original image (input to the preview generator). Settings below limit the size of the generated preview images (output). For more details see the xref:configuration/server/config_sample_php_parameters.adoc#define-the-maximum-dimensions-of-the-original-image-for-preview-generation[Define the maximum dimensions of the original image for preview generation].
```

**Review by mmattel — APPROVED** [review]
LGTM 👍

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
I rebased to trigger CI

**Comment by phil-davis** [comment]
Backport to 10.14 is PR #1304

**Pull request merged**